modules/representation_management/app/models/representation_management/form_2122_base.rb

```diff
@@ -107,5 +118,16 @@ def consent_limits_must_contain_valid_values
         end
       end
     end
+
+    def find_representative
+      AccreditedIndividual.find_by(id: representative_id) ||
+        Veteran::Service::Representative.find_by(representative_id:)
+    end
+
+    def representative_resolves
```

Comment:
Should `return unless find_representative.nil?` be `return unless representative.nil?`? Make use of `#find_representative` and the memoization?

Reply:
That makes sense!  I made the change here and for `organization_name` in the 2122 as well.